rfd/0027-mtls-metrics.md

```diff
@@ -0,0 +1,43 @@
+---
+author: Naji Obeid (naji@goteleport.com)
+state: draft
+---
+
+# RFD 27 - mTLS Prometheus metrics
+
+## What
+
+This RFD proposes the option of securing the `/metrics` endpoint read by prometheus with mTLS.
+
+## Why
+
+To prevent leaking metrics shipped to prometheus over an insecure network.
+
+## Details
```

Comment:
please also add migration steps - how does someone switch to an mTLS endpoint on an existing cluster